docs/API.md

```diff
@@ -1643,18 +1643,17 @@ composePromise
 
 <a name="selectObjectContent"></a>
 
-### selectObjectContent(bucketName, objectName, selectOpts[, callback])
+### selectObjectContent(bucketName, objectName, selectOpt)
```

Comment:
Very minor issue here, shouldn't this be `selectOpts` as the table below refers to `selectOpts`?

```suggestion
### selectObjectContent(bucketName, objectName, selectOpts)
```

Reply:
I have updated. thank you. :+1: